src/assets/scss/layout/_layout.scss

```diff
@@ -36,3 +36,18 @@ section {
     margin-right:1rem !important;
     top:-0.5 !important;
 }
+
+element.style {}
+
+.pf-c-nav__link,
+.pf-c-nav__link:hover,
+.pf-c-nav__link:focus,
+.pf-c-nav__link:active {
+    width: 100%;
+    text-decoration: none;
+    border: none;
+}
+
+.pf-c-nav__link {
+    min-height: 2rem;
```

Comment:
I ran your code and found out that this is the only rule which matters. The above rules applying to `.pf-c-nav__link, .pf-c-nav__link:hover, .pf-c-nav__link:focus, .pf-c-nav__link:active`, _seem_ to have no effect, looks the same if they're commented out. Correct me if I'm wrong... Do they serve a purpose?

Reply:
I just checked. No, they don't, my bad. I'm sending an update now.